osmoutils/accum/accum.go

```diff
@@ -65,6 +65,19 @@ func MakeAccumulatorWithValueAndShare(accumStore store.KVStore, accumName string
 	return setAccumulator(&newAccum, accumValue, totalShares)
 }
 
+// OverwriteAccumulatorUnsafe overwrites the accumulator with the given name in accumStore.
+// Use with caution as this method is only meant for use in migrations.
+func OverwriteAccumulatorUnsafe(accumStore store.KVStore, accumName string, accumValue sdk.DecCoins, totalShares osmomath.Dec) error {
+	if !accumStore.Has(formatAccumPrefixKey(accumName)) {
+		return errors.New("Accumulator with given name does not exist in store")
+	}
+
+	newAccum := AccumulatorObject{accumStore, accumName, accumValue, totalShares}
```

Comment:
Is there any sanity check we can/should do wrt totalShares here? Or not really?

Reply:
We could check if `accumValue` is non-negative or `totalShares` are not-zero but I don't think that the benefit of these checks would be greater than the added complexity.

IMO, keeping it as is fine. However, happy to implement checks if there are any specific suggestions